src/components/timeout/index.njk

```diff
@@ -7,6 +7,7 @@ title: Timeout
 This component may be deprecated as runner now does server side timeouts and this client side one is flaky.
 {{
     patternlibExample({
-        "path": "components/timeout/examples/timeout/index.njk"
+        "path": "components/timeout/examples/timeout/index.njk",
+        "iframeHeight": 500
```

Comment:
You missed one height param soz

Reply:
JB's fault

Reply:
![kratz](https://user-images.githubusercontent.com/8447446/52335312-01300d80-29fa-11e9-8028-c59c0a53783e.jpg)